js/imageeditor/shared/core/editor.ts

```diff
@@ -899,8 +924,9 @@ export class LayerManager {
 		}
 
 		this.active_layer = this.layers[0].container;
+		this.active_layer_id = this.layers[0].id;
```

Comment:
init_layers is the initial loading, so we always set the active layer to layer 0 here.